packages/desktop-client/bin/build-browser.js

```diff
@@ -0,0 +1,25 @@
+#!usr/bin/env node
```

Comment:
```suggestion
#!/usr/bin/env node
```

…or remove it entirely and ask people to use `node` to invoke it